### ISSUE TITLE
Download Country:IP map remotely

### ISSUE THREAD

**Issue by ignoramous** [celzero/rethink-app#222]
`dbip.v4` and `dbip.v6` add 5.1MB to the final APK, which could be avoided.

https://github.com/celzero/rethink-app/blob/b17f3850449713596932b9859c01f88d04ffa31f/app/src/main/java/com/celzero/bravedns/net/doh/CountryMap.java#L45-L48

Download these assets remotely, from `download.bravedns.com/ipdb/<tstamp>` instead?

**Comment by ignoramous** [comment]
See #105

**Comment by ignoramous** [comment]
Impl at `dl.rethinkdns.com` backed by Cloudflare R2.

**Comment by salmanaligeek** [comment]
Any update on it?

**Comment by ignoramous** [comment]
Not really, since this issue isn't high on our priority list. But someday, we will get to it.
